pkgs/tools/misc/kisslicer/default.nix

```diff
@@ -0,0 +1,55 @@
+{ fetchurl
+, libX11
+, mesa
+, makeWrapper
+, stdenv
+, unzip
+, inidir ? "\\\$HOME/.config/kisslicer"
+}:
+
+let
+  name = "kisslicer";
+  version = "1.6.2";
+
+  libPath = stdenv.lib.makeLibraryPath [
+    mesa
+    stdenv.cc.cc
+    libX11
+  ];
+
+in stdenv.mkDerivation rec {
+  inherit version name;
```

Comment:
The `name` should include the version. Best to replace this line with
```
  name = "kisslicer-1.6.2";
```
No need for a `version` field since it's not used anywhere.

Reply:
fixed.